Product/pom.xml

```diff
@@ -1132,6 +1133,11 @@
         <version>1.4.1</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
```

Comment:
We already have commons-fileupload artifact in this pom.xml. Just remove this block and update the existing entry.  In additional to that, the new entry has misspelling for commons-fileupload